data/json/items/gun/57.json

```
@@ -44,8 +44,17 @@
     "looks_like": "hk_mp5",
     "type": "GUN",
     "reload_noise_volume": 10,
-    "name": { "str": "FN P90" },
-    "description": "The first in a new genre of guns, termed \"personal defense weapons.\"  FN designed the P90 to use their proprietary 5.7x28mm ammunition.  It is made for firing bursts manageably.",
+    "name": { "str": "FN 5.7x28mm PDW" },
+    "description": "An FN manufactured personal defence weapon, this automatic firearm is chambered for the high-velocity proprietary 5.7x28mm cartridge.  Excepting top-mounted 50-round custom magazines which are fitted to the receiver of the weapon behind the grip, thus making it a bullpup design, this little submachine gun packs quite the nasty payload.  It’s comparatively small high-speed calibre makes it excellent at both defeating armoured opponents, and firing short bursts manageably.",
```

```suggestion
    "description": "An FN manufactured personal defence weapon, this automatic firearm is chambered for the high-velocity proprietary 5.7x28mm cartridge.  Accepting top-mounted 50-round custom magazines which are fitted to the receiver of the weapon behind the grip, thus making it a bullpup design, this little submachine gun packs quite the nasty payload.  It’s comparatively small high-speed calibre makes it excellent at both defeating armoured opponents, and firing short bursts manageably.",
```

---

Additoinal typo fix + game set in America has American spellings

```suggestion
    "description": "An FN manufactured personal defence weapon, this automatic firearm is chambered for the high-velocity proprietary 5.7x28mm cartridge.  Accepting top-mounted 50-round custom magazines which are fitted to the receiver of the weapon behind the grip, thus making it a bullpup design, this little submachine gun packs quite the nasty payload.  Its comparatively small high-speed caliber makes it excellent at both defeating armored opponents, and firing short bursts manageably.",
```